src/Workspaces/Core/Portable/CodeFixes/FixAllOccurrences/BatchFixAllProvider.cs

```diff
@@ -322,36 +320,25 @@ public virtual string GetFixAllTitle(FixAllState fixAllState)
             // More complex case.  We have multiple changes to the document.  Apply them in order
             // to get the final document.
 
-            var totalChangesIntervalTree = SimpleIntervalTree.Create(this);
```

Comment:
All this code was extracted, virtually unchanged, into DocumentTextChangeMerger.  That way we can share the same text merging algorithm and smarts between the Batch-Fixer and The CodeCleanup subsystem.

Reply:
If you look commit by commit you will see that.